requirements_all.txt

```diff
@@ -526,7 +526,7 @@ prometheus_client==0.0.19
 # homeassistant.components.sensor.systemmonitor
 psutil==5.4.0
 
-# homeassistant.components.wink
+# homeassistant.components.wink.wink
```

Comment:
@balloob I am not sure why I am getting an error on requirements all. When I run the script it adds both lines.

homeassistant.components.wink
homeassistant.components.wink.wink 

I believe since it is now located in a folder called wink I should only get the second one correct?